examples/features/nse-composition/README.md

```diff
@@ -71,8 +71,6 @@ spec:
           env:
             - name: NSM_NETWORK_SERVICES
               value: kernel://nse-composition/nsm-1
-            - name: NSM_REQUEST_TIMEOUT
-              value: 75s
```

Comment:
Should we increase CPU limit for nsmgr? As I remember you said this is required.

Reply:
Ah I see you did it